### PR TITLE
Reserve 16 enums for IMG.

### PR DESCRIPTION
**Pull request by Jeremy-Kemp**
Actual enum values will be filled in at a later date.

### CODE REVIEW

File: xml/cl.xml [hunk 1]
```diff
@@ -1719,8 +1719,12 @@ server's OpenCL/api-docs repository.
             <unused start="0x4213" end="0x421F"/>
     </enums>
 
-    <enums start="0x4220" end="0x4FFF" name="enums.4220" comment="Reserved for vendor extensions. Allocate in groups of 16.">
-            <unused start="0x420F" end="0x4FFF"/>
+    <enums start="0x4220" end="0x4230" name="enums.4220" vendor="IMG">
```
Comment (alycm) [inline xml/cl.xml:1722]: This reserves 17 values but reserving is done in blocks of 16.   If you actually need 17 values then reserve a block of size 32.

**Review by alycm — CHANGES_REQUESTED** [review]
Reserve 1 fewer or 15 more values.

File: xml/cl.xml (hunk 1 above)
Comment (Jeremy-Kemp) [reply xml/cl.xml]: Woops, thats embarrassing. Should be fixed now.

Comment (alycm) [reply xml/cl.xml]: Thanks!

**Review by alycm — APPROVED** [review]
LGTM, I'll just merge this immediately.

**Pull request merged**